src/main/java/com/kiki/userRegister/controller/response/UserResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.kiki.userRegister.controller.response;
```

Comment:
Javaではパッケージ名に大文字アルファベットは使わないです。
```suggestion
package com.kiki.userregister.controller.response;
```